package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
Yeah I was thinking about composition, something like
`parallel lint:watch (seq 'tsc -w ' 'serve:watch'))` but we `tsc -w` to emit something when it's done compiling so that the next step may start.

 Maybe something like [ts-watch](https://github.com/gilamran/tsc-watch) could solve this?